src/pages/Detail/Component/Recommend.js

```diff
@@ -0,0 +1,109 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+
+const Recommend = ({ bookData, setBookData }) => {
+  // * TODO :  `books/${bookData.sub_category}?count=10` --> 동일한 sub category에서 book data 10개
+  useEffect(() => {
+    fetch('/data/DetailData.json', { method: 'GET' })
+      .then(response => {
+        return response.json();
+      })
+      .then(result => {
```

Comment:
로직이 포함되지 않는다면 단순하게 () => () 형태로 작성해주세요!
```suggestion
      .then(response => response.json())
      .then(result => {
```